### PR TITLE
Changed development environment setup procedure

### PR DESCRIPTION
**Pull request by anishdixit-uw**
Don's process added in using pip install -e ooipy.

### CODE REVIEW

**Comment by anishdixit-uw** [comment]
@John-Ragland , this is ready to merge

File: docs/source/install_instructions.rst [hunk 1]
```diff
@@ -34,26 +34,26 @@ How to setup OOIPY development environment
...
+  conda create --name env_name pip
 
 3. Activate the environment using this command:
 
 .. code-block :: bash
 
   source activate env_name
 
-4. Run the script to setup the development environment and install dependencies using this command:
+4. Develop ooipy using this command in the same directory where the repository has been cloned:
```
Comment (John-Ragland) [inline docs/source/install_instructions.rst:45]: I think we can change this to:
4. install ooipy using this command in the same directory where the repository has been cloned.

**Comment by John-Ragland** [comment]
Closes #155

**Comment by John-Ragland** [comment]
Closes #154

File: docs/source/install_instructions.rst [hunk 2]
```diff
@@ -34,26 +34,26 @@ How to setup OOIPY development environment
...
 
 .. code-block :: bash
 
-  python ooipy/verify_setup.py
+  python
+  import ooipy
+  print(ooipy.__file__)
 
-If the path printed out matches your local installation path for the ooipy github repository, your development environment has been properly setup.
+If the path printed out matches the init.py from your local installation path for the ooipy github repository, your development environment has been properly setup.
```
Comment (lsetiawan) [inline docs/source/install_instructions.rst:59]: Should this be `init.py` or do you mean `__init__.py`?

Comment (anishdixit-uw) [reply docs/source/install_instructions.rst:59]: @lsetiawan, you are right, I will change it to the right convention

**Comment by anishdixit-uw** [comment]
> I don't see a step to install the [dev requirements](https://github.com/Ocean-Data-Lab/ooipy/blob/master/dev-requirements.txt)... should there be one?

@lsetiawan I am unsure about this, I thought `pip install -e ooipy ` would install those too.

If i am wrong, are you aware of what command would install the dev requirements? Would a `pip install -r dev_requirements.txt` work?

**Comment by John-Ragland** [comment]
> > I don't see a step to install the [dev requirements](https://github.com/Ocean-Data-Lab/ooipy/blob/master/dev-requirements.txt)... should there be one?
> 
> @lsetiawan I am unsure about this, I thought `pip install -e ooipy ` would install those too.
> 
> If i am wrong, are you aware of what command would install the dev requirements? Would a `pip install -r dev_requirements.txt` work?

Yeah, `pip install -r dev-requirements.txt` should work. I *think* that we will eventually use the features of hatch and the optional install stuff (which I still don't fully understand). But for this PR we should be good to just use this command.

**Comment by anishdixit-uw** [comment]
Thanks @lsetiawan , that helps!

File: docs/source/install_instructions.rst (hunk 1 above)
Comment (anishdixit-uw) [reply docs/source/install_instructions.rst:45]: Done

File: docs/source/install_instructions.rst (hunk 2 above)
Comment (anishdixit-uw) [reply docs/source/install_instructions.rst:59]: Done

**Comment by anishdixit-uw** [comment]
@John-Ragland , made all the changes, plz check and merge if it seems good!

**Review by John-Ragland — APPROVED** [review]

**Pull request merged**